templates/docs/examples/patterns/code-snippet/code-snippet.html

```diff
@@ -0,0 +1,29 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code Snippet{% endblock %}
+
+{% block content %}
+<div class="p-code-snippet">
+  <div class="p-code-snippet__header">
+    <h5 class="p-code-snippet__title">This is the code snippet title</h5>
```

Comment:
Would be good to have some more 'real' content for this (as we try in most examples).
Maybe some docs or tutorials would have something for "command" + "output" format.